### DISCUSSION TITLE
【如何嵌入B站视频】无法显示B站视频

### DISCUSSION THREAD

**Discussion by RevenSanchez** [tangly1024/NotionNext#2022] [General]
我使用这个项目嵌入B站视频但就是加载不出来
而如果嵌入油管视频就能够正常显示
![QQ截图20240302000111](https://github.com/tangly1024/NotionNext/assets/161605301/6d050b2e-18a2-4e2c-9224-ae41d26162c1)

**Comment by tangly1024** [comment]
Notion默认没有优化B站的视频；但您可以这样操作：
大致三步骤，复制B站的嵌入脚本，从脚本中复制视频的url，将url作为内嵌网页嵌入Notion。

1. 视频复制分享链接中的嵌入代码
![image](https://github.com/tangly1024/NotionNext/assets/15920488/0851a217-0334-4300-b1a4-709fafcf3862)

2. 转换代码为url格式：
默认复制的内容是 ：<iframe src="//player.bilibili.com/player.html?aid=1700928875&bvid=BV1aK42187hF&cid=1446312848&p=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
提取出其中src的部分：
![image](https://github.com/tangly1024/NotionNext/assets/15920488/2832155c-cdd9-4ca5-a68f-44a1502dc1d5)

即： //player.bilibili.com/player.html?aid=1700928875&bvid=BV1aK42187hF&cid=1446312848&p=1

3. 用Notion的embed嵌入
embed可以将任何网页无缝嵌入博客
![image](https://github.com/tangly1024/NotionNext/assets/15920488/5d6b9cbd-0980-45b6-8be4-07d437ef6496)

直接输入链接，并点击 Embed link 即可：
![image](https://github.com/tangly1024/NotionNext/assets/15920488/a3b15407-bd55-4a6f-89bd-6fd43716c95e)

# 效果

![image](https://github.com/tangly1024/NotionNext/assets/15920488/8c7f6510-7327-4e37-8a30-c6b0b46bedff)

**Comment by JennyChan2023** [comment]
"this embed couldn't be loaded"仍然显示这个
